tests/lib/rules/prop-types.js

```diff
@@ -1486,6 +1486,7 @@ ruleTester.run('prop-types', rule, {
         '  }',
         '}'
       ].join('\n'),
+      settings: {react: {flowVersion: '0.52.0'}},
```

Comment:
Is it better to have `{flowVersion: '0.52.0'}` be the default so the `eslint-plugin-react` upgrade is less likely to break things for people and not force them to have to add this?

Or, is it better to make `0.53.0` the default to encourage upgrades and easy adoption for new users?

Or, should we leave as is -- where there is no default and users have to make a conscious choice which they want to 
pick?

Any advice here @ljharb?